index.bs

```diff
@@ -345,6 +345,81 @@ and on when to use promises and when not to use promises,
 see <strong><a href="https://www.w3.org/2001/tag/doc/promises-guide">Writing
 Promise-Using Specifications</a></strong>.
 
+<h3 id="secure-context">Limit new features to secure contexts</h3>
+
+New capabilities added to the Web should be available only in
+<a href="https://w3c.github.io/webappsec-secure-contexts/">secure contexts</a>.
+Exposing them in non-secure contexts
+is discouraged and requires strong justification.
+The TAG is interested in hearing about and discussing cases
+where it is unclear whether exposing the capability
+in non-secure contexts is justifiable.
```

Comment:
How about just:
> The TAG is interested in hearing about and working to resolve any cases where exposure in non-secure contexts is being seriously considered.

Reply:
Or, simpler:
> The TAG is interested in hearing about cases 
where exposing new features in non-secure contexts is being considered.

(I'm pulling the "working to resolve" because I suspect that much of what we hear about might qualify as "not a new feature".)